fetch.bs

```diff
@@ -2133,6 +2133,19 @@ Unless stated otherwise, it is false.
 
 <p class=note>This flag is for exclusive use by HTML's render-blocking mechanism. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated boolean <dfn export for=request>https-upgrade</dfn>.
```

Comment:
"HTTPS upgrade" or "is HTTPS upgrade" seems more reasonable as a name. Not sure why it's all lowercase.

Reply:
Changed to "is HTTPS upgrade". Also changed "https-upgrade-fallback-url" to "HTTPS upgrade fallback URL", let me know if you prefer a different name.